mod.ts

```diff
@@ -2,22 +2,14 @@ export { MongoClient } from "./src/client.ts";
 export { Database } from "./src/database.ts";
 export { Collection } from "./src/collection/mod.ts";
 export * from "./src/types.ts";
-export * as Bson from "./deps.ts";
 export {
   Binary,
   BSONRegExp,
-  BSONSymbol,
-  Code,
-  DBRef,
   Decimal128,
   Double,
   Int32,
   Long,
-  MaxKey,
-  MinKey,
   ObjectId,
   Timestamp,
-  UUID,
-} from "./deps.ts";
-export type { Document } from "./deps.ts";
```

Comment:
Why remove these exports?

Reply:
i don't remember why i removed UUID but Document was removed since i put it in types.ts so it could get exported properly regardless of web-bson issues

Reply:
fixed in latest commit